internal/blobtesting/verify.go

```diff
@@ -16,6 +16,64 @@ import (
 	"github.com/kopia/kopia/repo/blob"
 )
 
+// VerifyTokenExpirationForGetBlob verifies that the token expiration
```

Comment:
Given that in the short term, this is only used for S3 with STS, let's move these verification functions and the corresponding assert to the s3 package itself.

Reply:
Sounds good. Done.